### PR TITLE
Q384 Add shuffleAnArray.py

### PR DESCRIPTION
**Pull request by peinanteng**

### CODE REVIEW

File: shuffleAnArray.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        return result
+
+    def shuffle(self):
+        """
+        Returns a random shuffling of the array.
+        :rtype: List[int]
+        """
+        result = self.reset()
+        length = len(result)
+        import random
```
Comment (nkukarl) [inline shuffleAnArray.py]: I don't think you should use `random` module.
If you can, then you can just do
```
random.shuffle(nums)
return nums
```

Comment (nkukarl) [inline shuffleAnArray.py]: Also, if you need to import a module, do it at the top of the file.
We strongly discourage local imports.
Sometimes, for circular dependency reasons, you have to do local imports.
Do it at the top of a certain function.

File: shuffleAnArray.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        :type nums: List[int]
+        """
+        self.nums = nums
+
+    def reset(self):
+        """
+        Resets the array to its original configuration and return it.
+        :rtype: List[int]
+        """
+        result = self.nums[:]
```
Comment (nkukarl) [inline shuffleAnArray.py]: Just do
```
return self.nums[:]
```

File: shuffleAnArray.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class Solution(object):
-
+    import random
```
Comment (nkukarl) [inline shuffleAnArray.py]: Not here, before class definition, very top.

Comment (peinanteng) [reply shuffleAnArray.py]: sure

**Pull request merged**